access_application.go

```diff
@@ -116,6 +117,26 @@ type SaasApplication struct {
 	CustomAttributes   []SAMLAttributeConfig `json:"custom_attributes,omitempty"`
 }
 
+type AppLauncherCustomization struct {
+	LandingPageDesign     LandingPageDesign `json:"landing_page_design"`
+	LogoURL               string            `json:"app_launcher_logo_url"`
+	HeaderBackgroundColor string            `json:"header_bg_color"`
+	BackgroundColor       string            `json:"bg_color"`
+	FooterLinks           []FooterLink      `json:"footer_links"`
+}
+
+type FooterLink struct {
+	Name string `json:"name"`
+	URL  string `json:"url"`
+}
+
+type LandingPageDesign struct {
```

Comment:
as this isn't namespaced, can we prefix this with `Access` to make it clear what the purpose is?